app/assets/stylesheets/v3/layout_components/pre-footer.scss

```diff
@@ -0,0 +1,102 @@
+.pre-footer {
+  @include grid-column(12,0);
+  background: $white;
+  height: $side-nav-height;
+  top: $top-bar-height;
+  z-index: 9000;
+
+  @include breakpoint(medium down) {
+    display: none;
+  }
```

Comment:
What is this section doing Sinead? I thought I understood it but I'm not noticing any content removed on small breakpoints.